src/Altinn.Platform/Altinn.Platform.Events/Events/Repository/PostgresRepository.cs

```diff
@@ -106,5 +187,45 @@ public async Task<List<CloudEvent>> Get(string after, DateTime? from, DateTime?
                 await _conn.CloseAsync();
             }
         }
+
+        /// <inheritdoc/>
+        public async Task<EventsSubscription> GetSubscription(int id)
+        {
+            EventsSubscription subscription = null;
+            try
+            {
+                await _conn.OpenAsync();
+
+                NpgsqlCommand pgcom = new NpgsqlCommand(getSubscriptionSql, _conn);
+                pgcom.Parameters.AddWithValue("_id", NpgsqlDbType.Integer, id);
+             
+                using (NpgsqlDataReader reader = pgcom.ExecuteReader())
+                {
+                    while (reader.Read())
+                    {
+                        subscription = new EventsSubscription();
+                        subscription.Id = Convert.ToInt32(reader["id"].ToString());
+                        subscription.SourceFilter = reader["sourcefilter"].ToString();
+                        subscription.SubjectFilter = reader["subjectfilter"].ToString();
+                        subscription.TypeFilter = reader["typefilter"].ToString();
+                        subscription.Consumer = reader["consumer"].ToString();
+                        subscription.EndPoint = reader["endpointurl"].ToString();
+                        subscription.CreatedBy = reader["createdby"].ToString();
+                        subscription.Created = DateTime.Parse(reader["time"].ToString());
+                    }
+                }
+
+                return subscription;
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine($" PostgresRepository // Get // Exception {JsonSerializer.Serialize(e)}");
```

Comment:
By using the notation below we will be able to get the exception out in a seperate column in the log in AI.                Console.WriteLine($" PostgresRepository // Get // Exception {ex}", JsonSerializer.Serialize(e));